Sources/MapboxNavigation/StylableButton.swift

```diff
@@ -60,3 +60,15 @@ open class CancelButton: Button {
 open class DismissButton: Button {
     
 }
+
+class BackButton: Button {
```

Comment:
Based on the screenshots, the back button seems to differ from the default back button in a NavigationViewController. The arrow icon has a shaft in addition to the arrowhead, and the “Back” text seems somewhat smaller than usual. I realize this is just an early iteration of the look and feel of this UI, but maybe we should find a way to align this button with the NavigationViewController back button, or at least make the button’s image and text size configurable.

Reply:
If you refer to `ResumeButton` that is used in `NavigationViewController`, then `BackButton` differs from it because it inherits from `Button`, whereas `ResumeButton` directly inherits from `UIControl`. I'll probably extract common logic to another class so that they look similar, we're more agile here because `BackButton` is internal for now.